File: fixtures/adversarial/T2/ts/moved-partly/after/src/padding.test.ts
import { format } from "./format";

describe("padding", () => {
  it("leaves an exact fit alone", () => {
    expect(format("abc", 3)).toBe("abc");
  });

  it("pads to the width", () => {
    const padded = format("a", 3);
    expect(padded).toBe("a  ");
    expect(format("ab", 3)).toBe("ab ");
  });

  it("cuts a long value down", () => {
    expect(format("abcde", 4)).toBe("abcd");
  });
});

File: fixtures/adversarial/T2/go/moved-partly/after/padding_test.go
package demo

import "testing"

func TestLeavesAnExactFitAlone(t *testing.T) {
	if got := FormatValue("abc", 3); got != "abc" {
		t.Errorf("an exact fit gave %q", got)
	}
}

func TestPadsToTheWidth(t *testing.T) {
	padded := FormatValue("a", 3)
	if padded != "a  " {
		t.Errorf("padding one character gave a padded %q", padded)
	}
	if got := FormatValue("ab", 3); got != "ab " {
		t.Errorf("padding two characters gave %q", got)
	}
}

func TestCutsALongValueDown(t *testing.T) {
	if got := FormatValue("abcde", 4); got != "abcd" {
		t.Errorf("truncating five characters gave %q", got)
	}
}
